.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Typically you don't commit editor-specific environment files. Not everyone on a team could be using vscode. To fix this, you could add `.vscode` to the `.gitignore` file

Reply:
added .vscode to .gitignore.